File: Back/src/main/java/com/showbook/back/entity/Shook.java
package com.showbook.back.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.AccessLevel;
import lombok.Builder;
import lombok.NoArgsConstructor;

@Entity
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Shook {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long shookId;

	@Column(nullable = false)
	private String bookTitle;

	@Column(nullable = false)
	private String shookImageUrl;

	@ManyToOne(fetch = FetchType.LAZY)
	@JoinColumn(name="book_id")
	private Book book;

	@ManyToOne(fetch = FetchType.LAZY)
	@JoinColumn(name="member_id")
	private Member member;

	@Builder
	public Shook(Long shookId, String bookTitle, String shookImageUrl, Book book, Member member) {
		this.shookId = shookId;
		this.bookTitle = bookTitle;
		this.shookImageUrl = shookImageUrl;
		this.book = book;
		this.member = member;
	}
}

File: ml/flask/models/Shook.py
from config.dbConfig import db

class Shook(db.Model) :
    '''
    shook_id : pk
    book_title : 슈욱 관련 책 제목
    shook_image_url : 슈욱 이미지 주소
    book_id : 슈욱 관련 책 id
    member_id : 슈욱 작성 멤버 id
    '''
    
    __tablename__ = "shook"
    
    shook_id = db.Column(db.Integer,nullable = False,autoincrement=True, primary_key=True)
    book_title = db.Column(db.String(255),nullable=False)
    shook_image_url = db.Column(db.Text(), nullable = False)
    book_id = db.Column(db.Integer,nullable = False)
    member_id = db.Column(db.Integer, nullable = False)